components/Header/MenuTab/index.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from "./styles.module.scss";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function MenuTab() {
+  const router = useRouter();
+  const { pathname } = router;
+
+  return (
+    <>
+      <Link
+        href="/community"
+        className={
+          pathname === "/community"
```

Comment:
```suggestion
        href="/boards"
        className={
          pathname === "/boards"
```
현재 폴더구조에서는 boards가 url이 맞을것같습니다!